src/doghouse/core/services/playback_service.py

```
@@ -0,0 +1,28 @@
+class PlaybackService:
+    """Service to run the delta engine against offline fixtures."""
+    
+    def __init__(self, engine: DeltaEngine):
+        self.engine = engine
+
+    def run_playback(self, playback_dir: Path) -> Tuple[Snapshot, Snapshot, Delta]:
+        """Run a delta comparison between baseline.json and current.json in the directory."""
+        baseline_path = playback_dir / "baseline.json"
+        current_path = playback_dir / "current.json"
+        
+        with open(current_path, "r") as f:
```

Ruff (0.15.7)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)